### PR TITLE
Draft: Add BatchNormalization operator

### LINKED ISSUE #5: Implement BatchNormalization operator
ONNX operator: https://github.com/onnx/onnx/blob/master/docs/Operators.md#BatchNormalization

I think no corresponding gorgonia op

### PR DESCRIPTION
**Pull request by Swopper050**
Closes #5

### CODE REVIEW

**Comment by Swopper050** [comment]
Implementation is still incomplete: needs train mode calculation as well.
However, as this option is probably rarely used I postponed the implementation, see #212.

File: ops/opset13/batch_normalization.go [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+	X := inputs[0]
+	scale := inputs[1]
+	B := inputs[2]
+	mean := inputs[3]
+	variance := inputs[4]
+
+	// We only support test mode, as this is by far the most common for inference models.
+	if !b.testMode {
+		return nil, ops.ErrUnsupportedAttribute("momentum", b)
+	}
```
Comment (wipsel) [inline ops/opset13/batch_normalization.go]: If we don't support "no test mode" would it not be better to catch this already in the init?

**Review by wipsel — CHANGES_REQUESTED** [review]

File: ops/opset13/batch_normalization.go (hunk 1 above)
Comment (Swopper050) [reply ops/opset13/batch_normalization.go]: good point
